### DISCUSSION TITLE
Can you migrate data from lazer to stable/beta?

### DISCUSSION THREAD

**Discussion by plotonnn** [ppy/osu#21533] [Q&A] [answered]
Half of what i download now is in osu lazer, so I wonder if I could just sync up the maps?

Because there are a lot of maps I didn't download on the beta version.

And I'd probably want them there.

**Comment by MishterKirby** [comment, marked as answer]
You can import your beatmaps from stable by going into the Maintenance section in the settings
 
![image](https://user-images.githubusercontent.com/48867925/205981384-1f6c6059-3db5-4fb0-a97a-9dd94ac7023d.png)

**Reply by plotonnn** [reply]
I'm sorry, maybe I put it wrong, I meant FROM osu!lazer TO beta

**Reply by MishterKirby** [reply]
Unfortunately, that's not possible, you can only transfer maps from stable to lazer but not the other way around

**Reply by plotonnn** [reply]
oh, okay, thanks!!!

**Comment by Hasster1** [comment]
You can use third party tools like https://github.com/kabiiQ/BeatmapExporter to export beatmaps from osu lazer into .osk files and then manually add them to the osu!stable
